### DISCUSSION TITLE
sfh2exp: when f_burst = 0.0, burst_age defaults to first value in list?

### DISCUSSION THREAD

**Discussion by mjrutkowski** [mboquien/cigale#106] [General]
When running fits to a set of galaxy SEDs with module sfh2exp, I allowed the set of "f_burst" values to vary of a range between 0 and 10%, explicitly including f_burst = 0.0.  The majority of galaxies were fit with f_burst > 0.0 as expected, but for a small set of galaxies when the best fit for f_burst==0.0, the burst_age reported is always the first in the set specified for this parameter in the pcigale.ini file.  The selection of the first value in this list is just "out of necessity" when constructing the results.fits file, correction?  That is, if the fraction is zero, any burst age is "valid" correct; a value for this burst age parameter must written and CIGALE simply takes the first from the list of specified parameters as a placeholder, correct?

**Comment by mboquien** [comment]
Yes, this is indeed the case. When several models have exactly the same likelihood, the first model is selected.

**Comment by mjrutkowski** [comment]
Perfect. Thanks so much !
Michael Rutkowski

On Sep 29, 2024, at 1:16 PM, Médéric Boquien ***@***.***> wrote:

﻿

CAUTION: This e-mail originated from outside the Minnesota State System. Only click links or open attachments from trusted sources. Please report suspicious messages using the "Report Message Button".

Yes, this is indeed the case. When several models have exactly the same likelihood, the first model is selected.
